File: totosuki/ABC/B/130.py
N, X = map(int, input().split())
L = list(map(int, input().split()))
D = [0]

for i in range(N):
  rslt = D[i] + L[i]
  D.append(rslt)

cnt = 0
for i in range(N+1):
  if D[i] <= X:
    cnt += 1

print(cnt)

File: OCH_Tao/ABC/B/130.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const [N, X] = (input[0].split(" ")).map(x => parseInt(x, 10));
const L = (input[1].split(" ")).map(x => parseInt(x, 10));
let tmp = [0];
for (const i of L) {
  tmp.push(tmp[tmp.length - 1] + i);
}
console.log((tmp.filter(x => x <= X)).length);